http-wrapper.md

```diff
@@ -5,15 +5,15 @@ It also allows us to send files if we so choose.
 Each node should have an HTTP server that listens for incoming messages.
 
 All messages sending data should use the `POST` action.
-For example, a `ping` would use `GET`, but `chat` or `nodelist` would use `POST`. 
+For example, a `ping` would use `GET`, but `chat` or `nodelist` would use `POST`.
 
 
 An example request could look something like this
 
 ```bash
 curl -H "Content-Type: application/json" \
      -X POST \
-     -d '{"message":"encrypted message"}' \
+     -d '{"message":"encrypted message","uid":"1234"}' \
```

Comment:
we need this so we can look up which public key to use.

thoughts/question:
Would it be a problem to just send the public key? - this would effectively make the public key the uid though